apps/api/src/app/shared/dtos/pagination-request.ts

```diff
@@ -2,10 +2,15 @@ import { ApiPropertyOptional } from '@nestjs/swagger';
 import { Type } from 'class-transformer';
 import { IsInt, Max, Min } from 'class-validator';
 
-export type Constructor<I> = new (...args: any[]) => I;
+import { Constructor } from '../types';
+
+export interface IPagination {
+  page?: number;
+  limit?: number;
```

Comment:
they are optional but have their defaults, meaning that during the dto transformation if the query param value doesn't exist it will be defined with defaults on dto... I'll update it